src/app/home/room-creation-modal/room-creation-modal.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { MatDialogRef } from "@angular/material/dialog";
+import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { GameType } from "../models/game-type";
+
+@Component({
+  selector: 'room-creation-modal',
+  templateUrl: './room-creation-modal.component.html',
+  styleUrls: ['./room-creation-modal.component.scss']
+})
+export class RoomCreationModalComponent {
+  formGroup: FormGroup = this.formBuilder.group({
+    'name': [''],
+    'numberOfPlayers': [1, Validators.min(1)],
+    'iterations': [1, Validators.min(1)],
+    'typeOfGame': [GameType.Winner]
+  });
```

Comment:
I have never worked with this. Are these the fabled "reactive forms"? This seems worth learning :)

Reply:
yep  :)
<img width="402" alt="image" src="https://user-images.githubusercontent.com/17585880/195030307-ba56b400-5f65-4490-8e9c-070f39447de0.png">